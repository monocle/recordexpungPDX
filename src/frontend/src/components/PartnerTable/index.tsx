import { useState } from "react";
import { HashLink as Link } from "react-router-hash-link";
import {
  Disclosure,
  DisclosureButton,
  DisclosurePanel,
} from "../../vendor/@reach/disclosure";

import currentPartners from "./currentPartners";

export interface Partner {
  details: string[][];
  name: string;
  area: string;
  instructions: string;
  contacts: string[];
  website: string;
}

function PartnerElement({
  partner,
  idx,
  activeIdx,
  handleChange,
}: {
  partner: Partner;
  idx: number;
  activeIdx: number;
  handleChange: (idx: number) => void;
}) {
  return (
    <li className="bt bw2 b--lightest-blue1">
      <Disclosure
        open={idx === activeIdx}
        id={idx}
        onChange={() => handleChange(idx)}
      >
        <DisclosureButton className="flex-ns w-100 relative navy hover-blue pv3 ph3">
          <span className="w-70 db pr3 mb2 mb0-ns">{partner.name}</span>
          <span className="w-30 pr3">{partner.area}</span>
          <span className="absolute top-0 right-0 pt3 ph3">
            {idx === activeIdx ? (
              <span aria-hidden="true" className="fas fa-angle-up"></span>
            ) : (
              <span aria-hidden="true" className="fas fa-angle-down"></span>
            )}
          </span>
        </DisclosureButton>
        <DisclosurePanel>
          <div className="bl bw2 f5 b--blue pv3 ph3 mb3 ml3">
            <ul className="list mb2">
              {partner.details.map((detail, idx2) => (
                <li key={idx2} className="flex-ns mb3">
                  <span className="w10rem db fw6 mr3">{detail[0]}</span>
                  <span>{detail[1]}</span>
                </li>
              ))}
            </ul>
            <p className="mw6 lh-copy mb3">
              The majority of court fees are subject to waiver for
              income-qualified individuals who complete the waiver form.{" "}
              <Link className="link hover-blue bb" to="/manual#filepaperwork">
                Learn More
              </Link>
            </p>
            <hr className="bt b--black-05 mb3" />
            <p className="fw6 mb3">{partner.instructions}</p>
            <ul className="list mb3">
              {partner.contacts.map((contact, idx3) => (
                <li key={idx3} className="mb3">
                  {contact}
                </li>
              ))}
            </ul>
            <a href={partner.website} className="link hover-blue bb">
              {partner.website}
            </a>
          </div>
        </DisclosurePanel>
      </Disclosure>
    </li>
  );
}

export default function PartnerTable({ partners = currentPartners }) {
  // Only one Disclosure should be opened at a particular time.
  // activeIdx === -1 means that all Disclosures are closed
  const [activeIdx, setActiveIdx] = useState(-1);

  const toggleDiscolsure = (newIdx: number) => {
    setActiveIdx(newIdx === activeIdx ? -1 : newIdx);
  };

  return (
    <div className="ba bw3 br3 b--lightest-blue1 bg-white mb4">
      <div className="flex items-center justify-between">
        <h3 className="f3 fw9 pv4 ph3">Partners</h3>
        <svg
          className="mr3"
          style={{ width: "55px", height: "40px" }}
          aria-hidden="true"
          viewBox="0 0 110 80"
          fill="none"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            fillRule="evenodd"
            clipRule="evenodd"
            d="M6.56 78.42L1.88 68.04l4.68-15.85L8.58 27.7 12.48.82h7.3l6.14 3 1.59 8.74 8.81 1.77 7.86-1.77 7.85 1.77 3.78-3.54h8.03l4.98-2.23 7.44-1.8h25.76l6.56 5.8-3.77 12.87-5.72 11.1 4.19 3.09-1.26 6.43v32.37H6.56z"
            fill="#D0E1F7"
          />
          <circle cx="26.76" cy="21.05" r="2.76" fill="#2B75D2" />
          <circle cx="33.67" cy="24.51" r="2.76" fill="#2B75D2" />
          <circle cx="79.67" cy="16.51" r="2.76" fill="#2B75D2" />
          <circle cx="28.14" cy="30.04" r="2.76" fill="#2B75D2" />
          <circle cx="35.05" cy="65.29" r="2.76" fill="#2B75D2" />
          <circle cx="19.85" cy="44.55" r="2.76" fill="#2B75D2" />
          <circle cx="55.79" cy="44.55" r="2.76" fill="#2B75D2" />
        </svg>
      </div>
      <ul className="list">
        {partners.map((partner, index) => (
          <PartnerElement
            partner={partner}
            key={index}
            idx={index}
            activeIdx={activeIdx}
            handleChange={toggleDiscolsure}
          />
        ))}
      </ul>
    </div>
  );
}
