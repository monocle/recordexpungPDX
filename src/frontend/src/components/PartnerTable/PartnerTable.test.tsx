import { BrowserRouter } from "react-router-dom";
import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import userEvent from "@testing-library/user-event";
import renderer from "react-test-renderer";

import PartnerTable from ".";

test("PartnerTable renders correctly on initial display", () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <PartnerTable />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});

const partners = [
  {
    details: [],
    name: "Portland Community College",
    area: "",
    instructions: "",
    contacts: ["Leni Tupper"],
    website: "",
  },
  {
    details: [],
    name: "Criminals Anonymous",
    area: "",
    instructions: "",
    contacts: ["hrcubbedge1776@gmail.com"],
    website: "",
  },
];

test("PartnerTable only shows the details of one partner at a time", async () => {
  const user = userEvent.setup();

  render(<PartnerTable partners={partners} />, {
    wrapper: BrowserRouter,
  });

  expect(screen.queryByText(/portland community college/i)).toBeVisible();
  expect(screen.queryByText(/criminals anonymous/i)).toBeVisible();
  expect(screen.queryByText(/leni/i)).not.toBeVisible();
  expect(screen.queryByText(/hrcubbedge1776/i)).not.toBeVisible();

  await user.click(screen.getByText(/portland community/i));
  expect(screen.queryByText(/leni/i)).toBeVisible();
  expect(screen.queryByText(/hrcubbedge1776/i)).not.toBeVisible();

  await user.click(screen.getByText(/criminals anonymou/i));
  expect(screen.queryByText(/leni/i)).not.toBeVisible();
  expect(screen.queryByText(/hrcubbedge1776/i)).toBeVisible();
});
