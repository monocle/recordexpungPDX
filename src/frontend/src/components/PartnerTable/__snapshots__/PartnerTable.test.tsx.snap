// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PartnerTable renders correctly on initial display 1`] = `
<div
  className="ba bw3 br3 b--lightest-blue1 bg-white mb4"
>
  <div
    className="flex items-center justify-between"
  >
    <h3
      className="f3 fw9 pv4 ph3"
    >
      Partners
    </h3>
    <svg
      aria-hidden="true"
      className="mr3"
      fill="none"
      style={
        Object {
          "height": "40px",
          "width": "55px",
        }
      }
      viewBox="0 0 110 80"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        clipRule="evenodd"
        d="M6.56 78.42L1.88 68.04l4.68-15.85L8.58 27.7 12.48.82h7.3l6.14 3 1.59 8.74 8.81 1.77 7.86-1.77 7.85 1.77 3.78-3.54h8.03l4.98-2.23 7.44-1.8h25.76l6.56 5.8-3.77 12.87-5.72 11.1 4.19 3.09-1.26 6.43v32.37H6.56z"
        fill="#D0E1F7"
        fillRule="evenodd"
      />
      <circle
        cx="26.76"
        cy="21.05"
        fill="#2B75D2"
        r="2.76"
      />
      <circle
        cx="33.67"
        cy="24.51"
        fill="#2B75D2"
        r="2.76"
      />
      <circle
        cx="79.67"
        cy="16.51"
        fill="#2B75D2"
        r="2.76"
      />
      <circle
        cx="28.14"
        cy="30.04"
        fill="#2B75D2"
        r="2.76"
      />
      <circle
        cx="35.05"
        cy="65.29"
        fill="#2B75D2"
        r="2.76"
      />
      <circle
        cx="19.85"
        cy="44.55"
        fill="#2B75D2"
        r="2.76"
      />
      <circle
        cx="55.79"
        cy="44.55"
        fill="#2B75D2"
        r="2.76"
      />
    </svg>
  </div>
  <ul
    className="list"
  >
    <li
      className="bt bw2 b--lightest-blue1"
    >
      <button
        aria-controls="panel--0"
        aria-expanded={false}
        className="flex-ns w-100 relative navy hover-blue pv3 ph3"
        data-reach-disclosure-button=""
        data-state="collapsed"
        onClick={[Function]}
      >
        <span
          className="w-70 db pr3 mb2 mb0-ns"
        >
          Portland Community College
        </span>
        <span
          className="w-30 pr3"
        >
          Northeast Portland
        </span>
        <span
          className="absolute top-0 right-0 pt3 ph3"
        >
          <span
            aria-hidden="true"
            className="fas fa-angle-down"
          />
        </span>
      </button>
      <div
        data-reach-disclosure-panel=""
        data-state="collapsed"
        hidden={true}
        id="panel--0"
      >
        <div
          className="bl bw2 f5 b--blue pv3 ph3 mb3 ml3"
        >
          <ul
            className="list mb2"
          >
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Locations
              </span>
              <span>
                Multnomah County
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Income Restrictions
              </span>
              <span>
                Low-income only
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Analysis Cost
              </span>
              <span>
                Free
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Paperwork Cost
              </span>
              <span>
                Free
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Court Fees
              </span>
              <span>
                Not Included
              </span>
            </li>
          </ul>
          <p
            className="mw6 lh-copy mb3"
          >
            The majority of court fees are subject to waiver for income-qualified individuals who complete the waiver form.
             
            <a
              className="link hover-blue bb"
              href="/manual#filepaperwork"
              onClick={[Function]}
            >
              Learn More
            </a>
          </p>
          <hr
            className="bt b--black-05 mb3"
          />
          <p
            className="fw6 mb3"
          >
            Contact
          </p>
          <ul
            className="list mb3"
          >
            <li
              className="mb3"
            >
              Leni Tupper
            </li>
          </ul>
          <a
            className="link hover-blue bb"
            href="https://www.pcc.edu/clear-clinic/intake-form/"
          >
            https://www.pcc.edu/clear-clinic/intake-form/
          </a>
        </div>
      </div>
    </li>
    <li
      className="bt bw2 b--lightest-blue1"
    >
      <button
        aria-controls="panel--1"
        aria-expanded={false}
        className="flex-ns w-100 relative navy hover-blue pv3 ph3"
        data-reach-disclosure-button=""
        data-state="collapsed"
        onClick={[Function]}
      >
        <span
          className="w-70 db pr3 mb2 mb0-ns"
        >
          Criminals Anonymous
        </span>
        <span
          className="w-30 pr3"
        >
          East Portland
        </span>
        <span
          className="absolute top-0 right-0 pt3 ph3"
        >
          <span
            aria-hidden="true"
            className="fas fa-angle-down"
          />
        </span>
      </button>
      <div
        data-reach-disclosure-panel=""
        data-state="collapsed"
        hidden={true}
        id="panel--1"
      >
        <div
          className="bl bw2 f5 b--blue pv3 ph3 mb3 ml3"
        >
          <ul
            className="list mb2"
          >
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Locations
              </span>
              <span>
                Multnomah County
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Income Restrictions
              </span>
              <span>
                None
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Analysis Cost
              </span>
              <span>
                Free
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Paperwork Cost
              </span>
              <span>
                $100/case
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Court Fees
              </span>
              <span>
                Not Included
              </span>
            </li>
          </ul>
          <p
            className="mw6 lh-copy mb3"
          >
            The majority of court fees are subject to waiver for income-qualified individuals who complete the waiver form.
             
            <a
              className="link hover-blue bb"
              href="/manual#filepaperwork"
              onClick={[Function]}
            >
              Learn More
            </a>
          </p>
          <hr
            className="bt b--black-05 mb3"
          />
          <p
            className="fw6 mb3"
          >
            Contact
          </p>
          <ul
            className="list mb3"
          >
            <li
              className="mb3"
            >
              hrcubbedge1776@gmail.com
            </li>
          </ul>
          <a
            className="link hover-blue bb"
            href="https://www.qiu-qiulaw.com/crimanon"
          >
            https://www.qiu-qiulaw.com/crimanon
          </a>
        </div>
      </div>
    </li>
    <li
      className="bt bw2 b--lightest-blue1"
    >
      <button
        aria-controls="panel--2"
        aria-expanded={false}
        className="flex-ns w-100 relative navy hover-blue pv3 ph3"
        data-reach-disclosure-button=""
        data-state="collapsed"
        onClick={[Function]}
      >
        <span
          className="w-70 db pr3 mb2 mb0-ns"
        >
          Qiu-Qiu Law
        </span>
        <span
          className="w-30 pr3"
        >
          Portland
        </span>
        <span
          className="absolute top-0 right-0 pt3 ph3"
        >
          <span
            aria-hidden="true"
            className="fas fa-angle-down"
          />
        </span>
      </button>
      <div
        data-reach-disclosure-panel=""
        data-state="collapsed"
        hidden={true}
        id="panel--2"
      >
        <div
          className="bl bw2 f5 b--blue pv3 ph3 mb3 ml3"
        >
          <ul
            className="list mb2"
          >
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Locations
              </span>
              <span>
                Portland Metro
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Income Restrictions
              </span>
              <span>
                None
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Analysis Cost
              </span>
              <span>
                Free
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Paperwork Cost
              </span>
              <span>
                $100/case
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Court Fees
              </span>
              <span>
                Not Included
              </span>
            </li>
          </ul>
          <p
            className="mw6 lh-copy mb3"
          >
            The majority of court fees are subject to waiver for income-qualified individuals who complete the waiver form.
             
            <a
              className="link hover-blue bb"
              href="/manual#filepaperwork"
              onClick={[Function]}
            >
              Learn More
            </a>
          </p>
          <hr
            className="bt b--black-05 mb3"
          />
          <p
            className="fw6 mb3"
          >
            Check your eligibility
          </p>
          <ul
            className="list mb3"
          >
            <li
              className="mb3"
            />
          </ul>
          <a
            className="link hover-blue bb"
            href="https://www.qiu-qiulaw.com/recordsponge"
          >
            https://www.qiu-qiulaw.com/recordsponge
          </a>
        </div>
      </div>
    </li>
    <li
      className="bt bw2 b--lightest-blue1"
    >
      <button
        aria-controls="panel--3"
        aria-expanded={false}
        className="flex-ns w-100 relative navy hover-blue pv3 ph3"
        data-reach-disclosure-button=""
        data-state="collapsed"
        onClick={[Function]}
      >
        <span
          className="w-70 db pr3 mb2 mb0-ns"
        >
          Clackamas Workforce Partnership
        </span>
        <span
          className="w-30 pr3"
        >
          Oregon City
        </span>
        <span
          className="absolute top-0 right-0 pt3 ph3"
        >
          <span
            aria-hidden="true"
            className="fas fa-angle-down"
          />
        </span>
      </button>
      <div
        data-reach-disclosure-panel=""
        data-state="collapsed"
        hidden={true}
        id="panel--3"
      >
        <div
          className="bl bw2 f5 b--blue pv3 ph3 mb3 ml3"
        >
          <ul
            className="list mb2"
          >
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Locations
              </span>
              <span>
                Portland Metro
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Income Restrictions
              </span>
              <span>
                Unknown
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Analysis Cost
              </span>
              <span>
                Free
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Paperwork Cost
              </span>
              <span>
                Unknown
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Court Fees
              </span>
              <span>
                Not Included
              </span>
            </li>
          </ul>
          <p
            className="mw6 lh-copy mb3"
          >
            The majority of court fees are subject to waiver for income-qualified individuals who complete the waiver form.
             
            <a
              className="link hover-blue bb"
              href="/manual#filepaperwork"
              onClick={[Function]}
            >
              Learn More
            </a>
          </p>
          <hr
            className="bt b--black-05 mb3"
          />
          <p
            className="fw6 mb3"
          >
            Contact
          </p>
          <ul
            className="list mb3"
          >
            <li
              className="mb3"
            >
              Amanda Wall
            </li>
          </ul>
          <a
            className="link hover-blue bb"
            href="https://www.clackamasworkforce.org"
          >
            https://www.clackamasworkforce.org
          </a>
        </div>
      </div>
    </li>
    <li
      className="bt bw2 b--lightest-blue1"
    >
      <button
        aria-controls="panel--4"
        aria-expanded={false}
        className="flex-ns w-100 relative navy hover-blue pv3 ph3"
        data-reach-disclosure-button=""
        data-state="collapsed"
        onClick={[Function]}
      >
        <span
          className="w-70 db pr3 mb2 mb0-ns"
        >
          Lane Public Defender Services
        </span>
        <span
          className="w-30 pr3"
        >
          Eugene
        </span>
        <span
          className="absolute top-0 right-0 pt3 ph3"
        >
          <span
            aria-hidden="true"
            className="fas fa-angle-down"
          />
        </span>
      </button>
      <div
        data-reach-disclosure-panel=""
        data-state="collapsed"
        hidden={true}
        id="panel--4"
      >
        <div
          className="bl bw2 f5 b--blue pv3 ph3 mb3 ml3"
        >
          <ul
            className="list mb2"
          >
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Locations
              </span>
              <span>
                Central Oregon
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Income Restrictions
              </span>
              <span>
                Unknown
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Analysis Cost
              </span>
              <span>
                Free
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Paperwork Cost
              </span>
              <span>
                Unknown
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Court Fees
              </span>
              <span>
                Not Included
              </span>
            </li>
          </ul>
          <p
            className="mw6 lh-copy mb3"
          >
            The majority of court fees are subject to waiver for income-qualified individuals who complete the waiver form.
             
            <a
              className="link hover-blue bb"
              href="/manual#filepaperwork"
              onClick={[Function]}
            >
              Learn More
            </a>
          </p>
          <hr
            className="bt b--black-05 mb3"
          />
          <p
            className="fw6 mb3"
          >
            Contact
          </p>
          <ul
            className="list mb3"
          >
            <li
              className="mb3"
            >
              Zara Lukens
            </li>
          </ul>
          <a
            className="link hover-blue bb"
            href="http://www.lanepds.org"
          >
            http://www.lanepds.org
          </a>
        </div>
      </div>
    </li>
    <li
      className="bt bw2 b--lightest-blue1"
    >
      <button
        aria-controls="panel--5"
        aria-expanded={false}
        className="flex-ns w-100 relative navy hover-blue pv3 ph3"
        data-reach-disclosure-button=""
        data-state="collapsed"
        onClick={[Function]}
      >
        <span
          className="w-70 db pr3 mb2 mb0-ns"
        >
          Redmond Housing Works
        </span>
        <span
          className="w-30 pr3"
        >
          Redmond
        </span>
        <span
          className="absolute top-0 right-0 pt3 ph3"
        >
          <span
            aria-hidden="true"
            className="fas fa-angle-down"
          />
        </span>
      </button>
      <div
        data-reach-disclosure-panel=""
        data-state="collapsed"
        hidden={true}
        id="panel--5"
      >
        <div
          className="bl bw2 f5 b--blue pv3 ph3 mb3 ml3"
        >
          <ul
            className="list mb2"
          >
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Locations
              </span>
              <span>
                Central Oregon
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Income Restrictions
              </span>
              <span>
                Unknown
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Analysis Cost
              </span>
              <span>
                Free
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Paperwork Cost
              </span>
              <span>
                Unknown
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Court Fees
              </span>
              <span>
                Not Included
              </span>
            </li>
          </ul>
          <p
            className="mw6 lh-copy mb3"
          >
            The majority of court fees are subject to waiver for income-qualified individuals who complete the waiver form.
             
            <a
              className="link hover-blue bb"
              href="/manual#filepaperwork"
              onClick={[Function]}
            >
              Learn More
            </a>
          </p>
          <hr
            className="bt b--black-05 mb3"
          />
          <p
            className="fw6 mb3"
          >
            Contact
          </p>
          <ul
            className="list mb3"
          >
            <li
              className="mb3"
            >
              Andy Hall
            </li>
          </ul>
          <a
            className="link hover-blue bb"
            href="https://www.facebook.com/OregonHousingWorks"
          >
            https://www.facebook.com/OregonHousingWorks
          </a>
        </div>
      </div>
    </li>
    <li
      className="bt bw2 b--lightest-blue1"
    >
      <button
        aria-controls="panel--6"
        aria-expanded={false}
        className="flex-ns w-100 relative navy hover-blue pv3 ph3"
        data-reach-disclosure-button=""
        data-state="collapsed"
        onClick={[Function]}
      >
        <span
          className="w-70 db pr3 mb2 mb0-ns"
        >
          Signs of Hope
        </span>
        <span
          className="w-30 pr3"
        >
          Medford
        </span>
        <span
          className="absolute top-0 right-0 pt3 ph3"
        >
          <span
            aria-hidden="true"
            className="fas fa-angle-down"
          />
        </span>
      </button>
      <div
        data-reach-disclosure-panel=""
        data-state="collapsed"
        hidden={true}
        id="panel--6"
      >
        <div
          className="bl bw2 f5 b--blue pv3 ph3 mb3 ml3"
        >
          <ul
            className="list mb2"
          >
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Locations
              </span>
              <span>
                Jackson County
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Income Restrictions
              </span>
              <span>
                None
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Analysis Cost
              </span>
              <span>
                Free
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Paperwork Cost
              </span>
              <span>
                $100/case
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Court Fees
              </span>
              <span>
                Not Included
              </span>
            </li>
          </ul>
          <p
            className="mw6 lh-copy mb3"
          >
            The majority of court fees are subject to waiver for income-qualified individuals who complete the waiver form.
             
            <a
              className="link hover-blue bb"
              href="/manual#filepaperwork"
              onClick={[Function]}
            >
              Learn More
            </a>
          </p>
          <hr
            className="bt b--black-05 mb3"
          />
          <p
            className="fw6 mb3"
          >
            Contact
          </p>
          <ul
            className="list mb3"
          >
            <li
              className="mb3"
            >
              Sarah Kolb
            </li>
            <li
              className="mb3"
            >
              hope@janekolb.com
            </li>
            <li
              className="mb3"
            >
              541-821-5577
            </li>
          </ul>
          <a
            className="link hover-blue bb"
            href="https://www.janekolb.com"
          >
            https://www.janekolb.com
          </a>
        </div>
      </div>
    </li>
    <li
      className="bt bw2 b--lightest-blue1"
    >
      <button
        aria-controls="panel--7"
        aria-expanded={false}
        className="flex-ns w-100 relative navy hover-blue pv3 ph3"
        data-reach-disclosure-button=""
        data-state="collapsed"
        onClick={[Function]}
      >
        <span
          className="w-70 db pr3 mb2 mb0-ns"
        >
          Probation and Parole, Community Justice Center
        </span>
        <span
          className="w-30 pr3"
        >
          Medford
        </span>
        <span
          className="absolute top-0 right-0 pt3 ph3"
        >
          <span
            aria-hidden="true"
            className="fas fa-angle-down"
          />
        </span>
      </button>
      <div
        data-reach-disclosure-panel=""
        data-state="collapsed"
        hidden={true}
        id="panel--7"
      >
        <div
          className="bl bw2 f5 b--blue pv3 ph3 mb3 ml3"
        >
          <ul
            className="list mb2"
          >
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Locations
              </span>
              <span>
                Southern Oregon
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Income Restrictions
              </span>
              <span>
                Unknown
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Analysis Cost
              </span>
              <span>
                Free
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Paperwork Cost
              </span>
              <span>
                Unknown
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Court Fees
              </span>
              <span>
                Not Included
              </span>
            </li>
          </ul>
          <p
            className="mw6 lh-copy mb3"
          >
            The majority of court fees are subject to waiver for income-qualified individuals who complete the waiver form.
             
            <a
              className="link hover-blue bb"
              href="/manual#filepaperwork"
              onClick={[Function]}
            >
              Learn More
            </a>
          </p>
          <hr
            className="bt b--black-05 mb3"
          />
          <p
            className="fw6 mb3"
          >
            Contact
          </p>
          <ul
            className="list mb3"
          >
            <li
              className="mb3"
            >
              Eric Guyer
            </li>
          </ul>
          <a
            className="link hover-blue bb"
            href="https://jacksoncountyor.org/community-justice"
          >
            https://jacksoncountyor.org/community-justice
          </a>
        </div>
      </div>
    </li>
    <li
      className="bt bw2 b--lightest-blue1"
    >
      <button
        aria-controls="panel--8"
        aria-expanded={false}
        className="flex-ns w-100 relative navy hover-blue pv3 ph3"
        data-reach-disclosure-button=""
        data-state="collapsed"
        onClick={[Function]}
      >
        <span
          className="w-70 db pr3 mb2 mb0-ns"
        >
          Pendleton Legal Aid Services of Oregon
        </span>
        <span
          className="w-30 pr3"
        >
          Pendleton
        </span>
        <span
          className="absolute top-0 right-0 pt3 ph3"
        >
          <span
            aria-hidden="true"
            className="fas fa-angle-down"
          />
        </span>
      </button>
      <div
        data-reach-disclosure-panel=""
        data-state="collapsed"
        hidden={true}
        id="panel--8"
      >
        <div
          className="bl bw2 f5 b--blue pv3 ph3 mb3 ml3"
        >
          <ul
            className="list mb2"
          >
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Locations
              </span>
              <span>
                Umatilla County
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Income Restrictions
              </span>
              <span>
                None
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Analysis Cost
              </span>
              <span>
                Free
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Paperwork Cost
              </span>
              <span>
                Free
              </span>
            </li>
            <li
              className="flex-ns mb3"
            >
              <span
                className="w10rem db fw6 mr3"
              >
                Court Fees
              </span>
              <span>
                Not Included
              </span>
            </li>
          </ul>
          <p
            className="mw6 lh-copy mb3"
          >
            The majority of court fees are subject to waiver for income-qualified individuals who complete the waiver form.
             
            <a
              className="link hover-blue bb"
              href="/manual#filepaperwork"
              onClick={[Function]}
            >
              Learn More
            </a>
          </p>
          <hr
            className="bt b--black-05 mb3"
          />
          <p
            className="fw6 mb3"
          >
            Contact
          </p>
          <ul
            className="list mb3"
          >
            <li
              className="mb3"
            >
              Arron Guevara
            </li>
            <li
              className="mb3"
            >
              arron.guevara@lasoregon.org
            </li>
          </ul>
          <a
            className="link hover-blue bb"
            href="https://www.facebook.com/pendletonlegalaid/"
          >
            https://www.facebook.com/pendletonlegalaid/
          </a>
        </div>
      </div>
    </li>
  </ul>
</div>
`;
